### PR TITLE
10599_add_security_headers

### PR DESCRIPTION
**Pull request by ozamani9**
*Issue #:* /bcgov/entity#10599

*Description of changes:*

Updated the nginx.conf file to include security header information (i.e. Content Security Policy, etc.)

### CODE REVIEW

**Review by rarmitag — APPROVED** [review]

File: nginx.conf [hunk 1]
```diff
@@ -36,7 +59,6 @@ http {
    }
 
    location @index {
-     add_header Cache-Control no-cache;
```
Comment (severinbeauvais) [inline nginx.conf:39]: @ozamani9 Why did you remove this?

Comment (ozamani9) [reply nginx.conf:39]: it was added in the header section instead.  so its like i moved it, not removed it.. does that make sense

image

Comment (severinbeauvais) [reply nginx.conf:39]: Yes, perfect.
